content/02.introduction.md

```diff
@@ -165,6 +165,18 @@ This excessive clotting in lung capillaries has been suggested to related to a d
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
+The presentation of COVID-19 infection can vary greatly among pediatric patients, and in some cases they might manifest in distinct ways from COVID-19 in adults. 
+[@doi:10.1001/jamapediatrics.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.
+Most of the children reviewed had mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose or were asymptomatic.
```

Comment:
```suggestion
Of the more than a thousand cases described, the most commonly reports were for mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose, while three children were reported to be asymptomatic.
Severe lower respiratory infection was described in only one of the pediatric cases reviewed.
```
The specific mention I found in the article says that these were "frequent clinical manifestations" -- but I might be missing something that supports the "most of the children" phrasing?
I was also thinking it might be worth adding a mention that they did find one severe case.